_layouts/Default.html

```diff
@@ -0,0 +1,171 @@
+            <a href="https://spacecubed.com/" target="_blank"><img src="assets/img/logospacecubed.png" alt="Spacecubed logo."></a>
+    </div>
+    {%endcapture%}
+    <!-- Contact Us Methods -->
+    {%capture contactUs %}
+    <div class="row justify-content-center">
+      <div class="col-lg-3 col-md-6 text-center">
+          <a href="mailto:geogeeks.perth@gmail.com" target="_blank">
+              <i class="fa fa-envelope fa-3x wow bounceIn"></i>
+              <p>Email</p>
+          </a>
+      </div>
+      <div class="col-lg-3 col-md-6 text-center">
+          <a href="https://geogeeks.slack.com" target="_blank">
+              <i class="fa fa-slack fa-3x wow bounceIn"></i>
+              <p>Slack</p>
+          </a>
+      </div>
+      <!-- hiding this until we fix our Twitter account -->
+      <!-- <div class="col-lg-3 col-md-6 text-center">
+          <a href="https://twitter.com/geogeeks_au" target="_blank">
+              <i class="fa fa-twitter fa-3x wow bounceIn" data-wow-delay=".2s"></i>
+              <p>@geogeeks_au</p>
+          </a>
+      </div> -->
+      <div class="col-lg-3 col-md-6 text-center">
+          <a href="https://github.com/geogeeks-au" target="_blank">
+              <i class="fa fa-github fa-3x wow bounceIn" data-wow-delay=".3s"></i>
+              <p>GitHub</p>
+          </a>
+      </div>
+  </div>
+  <!-- Mail Chimp -->
+  <div class="row justify-content-center py-2 my-2">
+    <div id="mc_embed_signup">
+      <form action="//geogeeks.us15.list-manage.com/subscribe/post?u=e913fe1e61659e8300b501f44&amp;id=f03cf9758a" method="post" id="mc-embedded-subscribe-form" name="mc-embedded-subscribe-form" class="validate" target="_blank" novalidate>
+        <div id="mc_embed_signup_scroll">
+          <h4 class="text-black">Subscribe to our mailing list for updates:</h4>
+          <input type="email" value="" name="EMAIL" class="email" id="mce-EMAIL" placeholder="email address" required>
+          <!-- real people should not fill this in and expect good things - do not remove this or risk form bot signups-->
+          <div style="position: absolute; left: -5000px;" aria-hidden="true"><input type="text" name="b_e913fe1e61659e8300b501f44_f03cf9758a" tabindex="-1" value=""></div>
+          <div class="clear"><input type="submit" value="Subscribe" name="subscribe" id="mc-embedded-subscribe" class="button"></div>
+        </div>
+      </form>
+    </div>
+  </div>
+  {% endcapture %}
+
+<!-- Upcoming events -->
+{% capture events %}
+  <div class="container">
+    {% if site.data.events.events %}
+      {% for event in site.data.events.events %}
+        <div class="row justify-content-center">
+          <div class="col-lg-8 text-left mb-3">
+            <h5 class="text-white mt-0">
+              {{event.name}}
+            </h5>
+            <div class="text-white-50 pl-5">
+              {% if event.description %}{{event.description}}</br></br>{% endif %}
```

Comment:
`</br>` should be `<br />`

Reply:
yes, the oldSite.html can be removed,  I had kept it more for a quick reference for what the old site had interms of content, and general layout

Reply:
I will add the license and the rename the readme file later

Reply:
PS, that is my other github profile